apps/api/.env.example

```
@@ -3,17 +3,22 @@ WBD_CF_CAPTCHA_ENABLED=false
+# Required when WBD_DB_ENGINE=turso-cloud. Use an organization-scoped Platform API
 # token and inject it through your deployment's secret manager. The API creates
 # all databases and mints short-lived, database-scoped credentials at runtime.
 WBD_TURSO_ORG=your-org-or-account-slug
 WBD_TURSO_PLATFORM_TOKEN=your-platform-api-token
 WBD_TURSO_GROUP=default
 WBD_TURSO_DATABASE_PREFIX=wbd
 
+# Required when WBD_DB_ENGINE=tursod. The tursod service must be reachable only
+# through a trusted network because its current HTTP API has no authentication.
+WBD_TURSOD_URL=http://127.0.0.1:3001
+WBD_TURSOD_AUTH_TOKEN=your-tursod-auth-token
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WBD_TURSOD_AUTH_TOKEN key should go before the WBD_TURSOD_URL key

(UnorderedKey)